keyboards/tetris/keymaps/default/keymap.c

```diff
@@ -0,0 +1,747 @@
+#define RGB_Layer_3_Base_Color 0,0,0
+#define RGB_Layer_4_Base_Color 0,0,0
+#define RGB_Layer_5_Base_Color 0,0,0
+#define RGB_Layer_6_Base_Color 64, 0, 64
+#define RGB_Layer_7_Base_Color 0, 0,0
+#define RGB_Layer_8_Base_Color 0,0,0
+#define RGB_Layer_9_Base_Color 0,0,0
+
+
+#define RGB_Caps_Color 6,50,50
+#define RGB_TAP_On_Color 0,128,0
+#define RGB_TAP_Off_Color 128,0,0
+#define RGB_TAP_Base_Color 0,0,0
+
+extern rgblight_config_t	rgblight_config;
+bool				NUMLAY_STATUS		= false;
+int				RGB_LAYER0_mode		= 6;
+bool				RGB_TAP_STATE		= false;
+static uint8_t			current_layer		= 0;
+static bool			has_layer_changed	= true;
+static bool			save_rgbmode		= true;
+static bool			caps			= false;
+
+/* Tap Dance function */
+void dance_cln_finished( qk_tap_dance_state_t* state, void* user_data )
+{
+  if ( state->count == 1 )
+  {
+    register_code( KC_LSFT );
+  } else {
+    layer_on( _SYMB );
+    #ifdef AUDIO_ENABLE
+      PLAY_SONG( tone_taps );
+    #endif
+  }
+}
+
+
+void dance_cln_reset( qk_tap_dance_state_t* state, void* user_data )
+{
+  if ( state->count == 1 )
+  {
+    unregister_code( KC_LSFT );
+  }else  {
+    uint8_t layer = biton32( layer_state );
+    if ( layer == _SYMB )
+    {
+      layer_off( _SYMB );
+      rgblight_mode( RGB_LAYER0_mode );
+    }
+  }
+}
+
+enum my_keycodes {
+  RGB_TAP = SAFE_RANGE,
+  KC_00
+};
+
+/* Tap Dance Declarations */
+enum {
+  TD_Mute_Next = 0,
+  TD_SFT,
+  TD_LBRC,
+  TD_RBRC
+};
+/* Tap Dance Definitions */
+qk_tap_dance_action_t tap_dance_actions[] = {
+  [TD_Mute_Next]  = ACTION_TAP_DANCE_DOUBLE( KC_MUTE, KC_MNXT )
+  ,[TD_SFT]  = ACTION_TAP_DANCE_FN_ADVANCED( NULL, dance_cln_finished, dance_cln_reset )
+  ,[TD_LBRC]  = ACTION_TAP_DANCE_DOUBLE( KC_LBRC,   KC_LCBR )
+  ,[TD_RBRC]  = ACTION_TAP_DANCE_DOUBLE( KC_RBRC,   KC_RCBR )
+};
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/* BASE - QWERTY
+ *                       ,-----------------------------------------------------------------------------------.
+ *                       | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  |  BS  |  
+ *                       |------+------+------+------+------+-------------+------+------+------+------+------|
+ * shift + Esc = ~ -->   | Esc  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  | Enter|  <-- shift + Enter = "
+ *                       |------+------+------+------+------+------|------+------+------+------+------+------|
+ *                       | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  | Mute |  
+ *                       |------+------+------+------+------+------+------+------+------+------+------+------|  <-- tap: 1.Mute 2.Next
+ *                       | Ctrl | GUI  |  Alt |   [  |   -  |    Space    |   =  |   ]  |  Fx  |   \  |  Del |
+ *                       `-----------------------------------------------------------------------------------'
+ */
+[_BASE] = KEYMAP(
```

Comment:
Also, could you change all of the `KEYMAP`s to `LAYOUT` (or maybe even `LAYOUT_planck_mit`)?